elephant/sta.py

```diff
@@ -314,3 +320,57 @@ def spike_field_coherence(signal, spiketrain, **kwargs):
 
     return (pq.Quantity(sfc, units=pq.dimensionless),
             pq.Quantity(frequencies, units=pq.Hz))
+
+
+def stPR(spiketrains, bin_size=1*pq.ms, sigma=5*pq.ms):
+    """
+    Spike triggered population response as described in [1].
+
+    Parameters
+    ----------
+    spiketrains : list of neo.SpikeTrains
```

Comment:
of neo.SpikeTrain, not 's
it's counterintuitive but the static parses, that are in most python IDEs, don't know anything about the neo.SpikeTrains because there is no such class.

Reply:
fixed